controllers/binding/snapshotenvironmentbinding_adapter.go

```diff
@@ -73,7 +75,14 @@ func (a *Adapter) EnsureIntegrationTestPipelineForScenarioExists() (controller.O
 	}
 
 	if gitops.HaveBindingsFailed(a.snapshotEnvironmentBinding) {
-		a.logger.Info("The SnapshotEnvironmentBinding has failed to deploy on ephemeral environment and will be deleted later.", "snapshotEnvironmentBinding.Name", a.snapshotEnvironmentBinding.Name)
+		reasonMsg := "Unknown reason"
+		if conditionStatus := gitops.GetBindingConditionStatus(a.snapshotEnvironmentBinding); conditionStatus != nil {
+			reasonMsg = fmt.Sprintf("%s (%s)", conditionStatus.Reason, conditionStatus.Message)
+		}
+		a.logger.Info("The SnapshotEnvironmentBinding has failed to deploy on ephemeral environment and will be deleted later.",
+			"snapshotEnvironmentBinding.Name", a.snapshotEnvironmentBinding.Name,
+			"reason", reasonMsg,
```

Comment:
In case the conditionStatus is nill, do we have some clue at least where to look for more information regarding the failed deployment of SEB? I would add it to the error message since we will end up with **Unknown reason**

Reply:
SEB may not even exist at that time due prunning